### PR TITLE
update documentation for the repo

### PR DESCRIPTION
**Pull request by Omarzintan**
This PR updates the README.md and the CONTRIBUTING.md files for bumblebee-ai. 

# Main Changes
* README.md
  * update wording for `about`
  * take out list of features. (I think it might be better to have a separate `FEATURES.md` file if we want to talk about features)
  * take out starred features. (might bring this back if we think it is very necessary)
* CONTRIBUTING.md
  *   update example to use `bumblebee_api.ask_question`
  *  add `run_by_input_list` feature in the `More info about Bumblebee api` section.

### CODE REVIEW

**Review by RexfordEssilfie — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -368,12 +364,23 @@ def get_name(self):
...
-def run_by_tags(self, feature_tags: list):
+def run_by_tags(self, feature_tags: list, argmuments_list: list = []):
     """
-    Run a list of feature actions given their tags.
-    Arguments: <list> feature_tags
+    Run a list of feature actions given their tags and arguments lists.
+    Arguments: <list> feature_tags, <list<list>> arguments_list
     Returns: whatever the feature ran will return
-    Example: bumblebee_api.run_by_tags([feature_1_tag, feature_2_tag, feature_3_tag])
+    Example:
```
[the remark this answers is not in the document]
Comment (Omarzintan) [reply CONTRIBUTING.md:372]: You mean have more descriptive features?

[the remark this answers is not in the document]
Comment (RexfordEssilfie) [reply CONTRIBUTING.md:372]: Yep. Using real feature names and arguments.

[the remark this answers is not in the document]
Comment (Omarzintan) [reply CONTRIBUTING.md:372]: Ahh that's a good suggestion. I will do that and update this!

**Pull request merged**